case_studies/pikas/pikas.slim

```diff
@@ -1,116 +1,130 @@
 		"ELEVATION_RANGE", c(7539, 13507) * 3.048e-4 // elevation range in ft from map legend -> converting to km
 		);
 	
 	// Set up parameters with a user-defined function
+	// BCH: changes here are just to match minimal.slim; not claiming the changes are any better than the original :->
 	setupParams(defaults);
 	
 	defineConstant("FECUN", 1 / LIFETIME);
-	defineConstant("RHO", FECUN/((1+FECUN) * K)); // constant in spatial competition function
+	defineConstant("RHO", FECUN / ((1 + FECUN) * K));
 	defineGlobal("PARAMS", defaults);
-
+	
 	setSeed(SEED);
 	
+	initializeMutationRate(MU);
 	initializeMutationType("m1", 0.5, "f", 0.0);
 	initializeGenomicElementType("g1", m1, 1.0);
 	initializeGenomicElement(g1, 0, L-1);
-	initializeMutationRate(MU);
 	initializeRecombinationRate(R);
 	
 	// elevation params
+	// BCH: I'd suggest using all caps for all defined globals and constants
+	// BCH: I notice the map has only a handful of discrete values.  Any chance we can get better (more fine-grained) elevation data?  Not a big deal, just would be nice... But maybe we're matching an existing paper/model here?  I forget...
 	defineConstant("elevation_map", Image(MAP_FILE));
 	defineGlobal("Elevation", elevation_map.floatK * (ELEVATION_RANGE[1]-ELEVATION_RANGE[0]) + ELEVATION_RANGE[0]);
 	defineGlobal("Temperature", -10 * Elevation + 37); // estimate temperature from elevation (Collados-Lara AJ et al., 2020)
 	
-	// spatial competition
-	initializeInteractionType(1, "xy", reciprocal=T, maxDistance=SX * 3);
+	// spatial interaction for local density measurement (competition)
+	initializeInteractionType(1, "xy", reciprocal=T, maxDistance=3 * SX);
 	i1.setInteractionFunction("n", 1, SX);
 	
-	// mate choice
-	initializeInteractionType(2, "xy", reciprocal=T, maxDistance=SM * 3);
+	// spatial interaction for mate choice
+	initializeInteractionType(2, "xy", reciprocal=T, maxDistance=3 * SM);
 	i2.setInteractionFunction("n", 1, SM);
 }
 
 1 first() {
 	sim.addSubpop("p1", asInteger(K * WIDTH * HEIGHT));
 	p1.setSpatialBounds(c(0, 0, WIDTH, HEIGHT));
-	p1.defineSpatialMap("elevation", "xy", elevation_map.floatK, interpolate=T, valueRange=c(0,1), colors=c("#0000FF", "#FFFFFF")); // this map is (only) for visualizing elevation in the GUI
+	
+	// this map is (only) for visualizing elevation in the GUI
+	p1.defineSpatialMap("elevation", "xy", elevation_map.floatK, interpolate=T, valueRange=c(0,1), colors=c("#0000FF", "#FFFFFF"));
+	
+	// this map is actually used by the model
+	// BCH: I notice that, because of the way the color scale is defined, you can't really see the warming temperatures
+	// BCH: in the GUI well, apart from the whole map just turning blue.  Would be good to make the valueRange here
+	// BCH: broader, and define colors for different temperature ranges even outside of the initial range.  Would be nice
+	// BCH: if, e.g., the map slowly turned from blue/white to shades of orange and red, or something...?
```

Comment:
I see what you mean. Something has changed, because it used to serve as a better "movie" of changing temperature.

Will work on this